src/Samples/SarifTrim/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+
+namespace SarifTrim
+{
+    /// <summary>
+    ///  SarifTrim is a one-off sample removing some redundant and some excessively large parts of a Sarif log
+    ///  for handling of a huge log.
+    /// </summary>
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            if (args.Length < 1)
+            {
+                Console.WriteLine(@"Usage: SarifTrim [inputFilePath] [outputFilePath?] [removeParts?]
+    removeParts: Comma-separated sections to remove. Options: UriBaseIds,CodeFlows,RelatedLocations,Graphs,GraphTraversals,Stacks,WebRequests,WebResponses");
+                return;
+            }
+
+            string inputFilePath = args[0];
+            string outputFilePath = (args.Length > 1 ? args[1] : Path.ChangeExtension(inputFilePath, $"trim{Path.GetExtension(inputFilePath)}"));
```

Comment:
>trim [](start = 103, length = 4)

maybe use 'trimmed' here, so that a file such as MyLog.sarif is renamed to MyLog.trimmed.sarif?